packages/webpack/mixin.core.js

```diff
@@ -189,14 +188,7 @@ class WebpackMixin extends Mixin {
         this.configureCommand({
           command: 'develop',
           describe: `Serve ${name} in watch mode`,
-          builder: {
-            static: {
-              alias: 's',
-              default: false,
-              describe: 'Statically build locations',
-              type: 'boolean',
-            },
-          },
+          builder: {},
```

Comment:
I think you can get rid of this line completely.

Reply:
I would rather not: keeping it around will make the `develop` command easier to extend: if other mixins were to add arguments to it, one would otherwise always have to check if `builder` already exists.